executor.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal EnableDelayedExpansion
+
+if not defined VisualStudioVersion (
+    if defined VS140COMNTOOLS (
+        call "%VS140COMNTOOLS%\VsDevCmd.bat"
+        goto :Run
+    )
+
+    if defined VS120COMNTOOLS (
```

Comment:
We already require VS 2015 for anything to work so we can remove the VS12 fallback.